### DISCUSSION TITLE
There is a problem with the page style

### DISCUSSION THREAD

**Discussion by libi** [community/community#19681] [Other Feature Feedback, Questions, & Ideas] [answered]
There is a problem with the button position when the page width is small
![button](https://user-images.githubusercontent.com/7769922/176421639-3f31803c-f537-4dd6-9f29-f0b90bf2447d.png)

**Comment by melvincwng** [comment, marked as answer]
@libi 你好👋🏻，对我来说看起来像是一个 CSS 问题。

 对于错误，请提出票并将其报告给 GitHub Bug Report - https://support.github.com/contact/bug-report

 谢谢😄

 如果此回复对您有所帮助，请“标记为答案”或在 GitHub 上关注我。 感谢支持。
